extension/README.md

```diff
@@ -0,0 +1,117 @@
+# Front-end integration guide
+Adyen documentation for common workflow: https://docs.adyen.com/developers/checkout/api-integration
+
+## Requirements for CTP project:
+All the requirements above should be automatically created by the Adyen-integration. It will not create if there
+are already resources with same key in the CTP project.
+1. [API Extension subscription to Adyen-integration endpoints](./resources/api-extensions.json)
+1. [Custom types for payments](./resources/payment-custom-types.json)
+1. [Custom types for interface interactions](./resources/payment-interface-interaction-types.json)
```

Comment:
I'd avoid putting code references in the readme. If we move them the links will break.

Reply:
I will change this file in the next PR, but if you move the files, then of course you need to change the README as well? Same as if you rename some variables or you change the logic, you also need to update README?